jsdojo/3-5/script.js

```diff
@@ -0,0 +1,7 @@
+const date = new Date();
+const time = date.getHours();
+if ((time >= 9 && time <= 10) || (time >= 3 && time <= 4)) {
```

Comment:
> **Date.prototype.getHours()**
> An integer number, between 0 and 23, representing the hour for the given date according to local time.
[Date.prototype.getHours() - JavaScript | MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getHours#Return_value)

`Date.prototype.getHours()` returns an integer between 0 and 23. 
`(time >= 3 && time <= 4)` will be true between 3**am** and 4**am**.